### PR TITLE
[pixelfed] add guide for pixelfed

### LINKED ISSUE #695: Guide for pixelfed
[Pixelfed](https://pixelfed.org) is an decentralized image sharing platform with modern user interface similiar to instagram.

I'll try to setup an instance and write down a guide to bring the fediverse closer to uberspace.

**Help and tips wanted!**

### PR DESCRIPTION
**Pull request by taddydevil**
Pixelfed is a free, decentralized and ethical photo sharing platform, powered by ActivityPub federation. It comes with an modern user interface similiar to instagram.

closes #695

### CODE REVIEW

File: source/guide_pixelfed.rst [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+ * ``media:optimize`` - Finds any un-optimized media and performs optimization
+ * ``media:gc`` - Finds any media not used in statuses older than 1 hour and deletes them
+ * ``horizon:snapshot`` - Generates Horizon analytics snapshot
+ * ``story:gc`` - Finds expired Stories and deletes them
+
+Add the following cronjob to your :manual:`crontab <daemons-cron>` to run the scheduler every minute:
+
+.. code-block:: none
+
+ * * * * * cd /var/www/virtual/isabell/pixelfed && php artisan schedule:run >> /dev/null 2>&1
```
Comment (luto) [inline source/guide_pixelfed.rst:308]:
```suggestion
 * * * * * cd /var/www/virtual/$USER/pixelfed && php artisan schedule:run >> /dev/null 2>&1
```

**Comment by luto** [comment]
awesome! thank you very much :)

**Comment by luto** [comment]
@taddydevil feel free to order two cups ;)

**Comment by taddydevil** [comment]
@luto Do you mean for Prosody and Pixelfed 2 cups? 😅 Or overall 2 with the Pleroma Guide?

**Comment by luto** [comment]
one cup for each guide :)

**Pull request merged**